binding.gyp

```diff
@@ -2,18 +2,21 @@
   "targets": [
     {
       "target_name": "tree_sitter_swift_binding",
+      "dependencies": [
+        "<!(node -p \"require('node-addon-api').targets\"):node_addon_api_except",
+      ],
       "include_dirs": [
-        "<!(node -e \"require('nan')\")",
-        "src"
+        "src",
       ],
       "sources": [
         "bindings/node/binding.cc",
         "src/parser.c",
-        "src/scanner.c"
+        # NOTE: if your language has an external scanner, add it here.
+        "src/scanner.c",
```

Comment:
This was important.

Reply:
I think the issue I'm running into is that re-running `tree-sitter generate` removes this line, didn't realize this is part of the autogen. I haven't been able to figure out how to get the `generate` command to include the external scanner.